main.star

```diff
@@ -1,123 +1,65 @@
-input_parser = import_module("./src/package_io/input_parser.star")
 ethereum_package = import_module("github.com/ethpandaops/ethereum-package/main.star")
 contract_deployer = import_module("./src/contracts/contract_deployer.star")
-static_files = import_module(
-    "github.com/ethpandaops/ethereum-package/src/static_files/static_files.star"
-)
-participant_network = import_module("./src/participant_network.star")
-blockscout = import_module("./src/blockscout/blockscout_launcher.star")
-
-
-def get_l1_stuff(all_l1_participants, l1_network_params):
-    env_vars = {}
-    env_vars["L1_RPC_KIND"] = "any"
-    env_vars["WEB3_RPC_URL"] = str(all_l1_participants[0].el_context.rpc_http_url)
-    env_vars["L1_RPC_URL"] = str(all_l1_participants[0].el_context.rpc_http_url)
-    env_vars["CL_RPC_URL"] = str(all_l1_participants[0].cl_context.beacon_http_url)
-    env_vars["L1_CHAIN_ID"] = str(l1_network_params.network_id)
-    env_vars["L1_BLOCK_TIME"] = str(l1_network_params.seconds_per_slot)
-    env_vars["DEPLOYMENT_OUTFILE"] = (
-        "/workspace/optimism/packages/contracts-bedrock/deployments/"
-        + str(l1_network_params.network_id)
-        + "/kurtosis.json"
-    )
-    env_vars["STATE_DUMP_PATH"] = (
-        "/workspace/optimism/packages/contracts-bedrock/deployments/"
-        + str(l1_network_params.network_id)
-        + "/state-dump.json"
-    )
-
-    return env_vars
+static_files = import_module("github.com/ethpandaops/ethereum-package/src/static_files/static_files.star")
+l2_launcher = import_module("./src/l2.star")
+input_parser = import_module("./src/package_io/input_parser.star")
 
 
-def run(plan, args={}):
-    """Deploy a Optimism L2 with a local L1.
+def run(plan, args): 
+    """Deploy Optimism L2s on an Ethereum L1.
 
     Args:
-        args(yaml): Configures other aspects of the environment.
+        args(json): Configures other aspects of the environment.
```

Comment:
currently, our package indexer only picks up `json` (you can still pass it in as yaml but starlark converts it to json) so in order for the package to show up in package catalog, has to be set to json instead of yaml. I can update package indexer to support yaml type as well

Reply:
which package indexer?